From: Nathan Rossi <nathan@nathanrossi.com>
Subject: [PATCH] ARM: zynq: Fix up the target file name for u-boot

This should be 'u-boot.img' because all zynq configs are OF_EMBED.

Signed-off-by: Nathan Rossi <nathan@nathanrossi.com>
Upstream-Status: Configuration [mirror upstream...]
---
 include/configs/zynq-common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/zynq-common.h b/include/configs/zynq-common.h
index e5f2fc5..10c9f18 100644
--- a/include/configs/zynq-common.h
+++ b/include/configs/zynq-common.h
@@ -501,7 +501,7 @@
 #define CONFIG_SYS_MMCSD_FS_BOOT_PARTITION     1
 #define CONFIG_SPL_LIBDISK_SUPPORT
 #define CONFIG_SPL_FAT_SUPPORT
-#define CONFIG_SPL_FS_LOAD_PAYLOAD_NAME     "u-boot-dtb.img"
+#define CONFIG_SPL_FS_LOAD_PAYLOAD_NAME     "u-boot.img"
 #endif
 
 /* Disable dcache for SPL just for sure */
-- 
2.6.2

